pkgs/by-name/vd/vdhcoapp/package.nix

```diff
@@ -63,6 +62,9 @@ buildNpmPackage rec {
 
   postInstall = ''
     $out/bin/vdhcoapp install
+    mkdir -p $out/lib/mozilla/native-messaging-hosts
+    cp -rf /build/.mozilla/native-messaging-hosts/net.downloadhelper.coapp.json $out/lib/mozilla/native-messaging-hosts/net.downloadhelper.coapp.json
```

Comment:
like elaborated in my comment here https://github.com/NixOS/nixpkgs/pull/300051#discussion_r1564635579 we can not rely on running `vdhcoapp install` as it will break reproducibility and hydra will likely not have a firefox running, thus not create a manifest for firefox.

Reply:
yep i understood :) thanks for the guidance